src/codeu/chat/util/TextValidator.java

```diff
@@ -9,6 +9,7 @@
 public final class TextValidator {
 
   private static final Pattern validUserNamePattern = Pattern.compile("[a-zA-Z0-9]+");
+  private static final Pattern validPasswordPattern = Pattern.compile("[a-zA-Z0-9]+");
```

Comment:
`private static final` is effectively a constant and constants have names like such
`VALID_PASSWORD_PATTERN`